src/types/change.ts

```diff
@@ -64,6 +64,9 @@ export type ReferenceAttrs = Omit<InsertDeleteAttrs, 'operation'> & {
 
 export type NodeMoveAttrs = Omit<InsertDeleteAttrs, 'operation'> & {
   operation: CHANGE_OPERATION.move
+  indentationType?: 'indent' | 'unindent'
+  indentationNodeType?: 'section' | 'paragraph'
```

Comment:
in Ahmad's PR we managed to avoid direct reference to specific node types and it looks to me like it's going to be possible to derive it from the change itself. It would be preferable to keep TC plugin unaware of schema details.